larpandora/LArPandoraEventBuilding/LArPandoraShower/Tools/pandorashowertools.fcl

```diff
@@ -48,6 +48,17 @@ showernumelectronsenergy:{
      ShowerBestPlaneOutputLabel: "ShowerBestPlane"
  }
 
+showerestarenergy:{
+     tool_type:  ShowerESTAREnergy
+     BaseTools: @local::showerbasetools
+     CalorimetryAlg: @nil
+     # PFParticleLabel: "pandora"
+     ShowerEnergyOutputLabel: "ShowerEnergy"
+     ShowerBestPlaneOutputLabel: "ShowerBestPlane"
+     SCECorrectEField: false
+     ESTARFname: "ShowerEnergyReco/ESTAREnergyLookupCurve.root"
```

Comment:
This should probably be nullified here as it emphasises that each experiment needs to configure it.